### PR TITLE
[mesa] Update freedreno python script to fix a warning of the header

### PR DESCRIPTION
**Pull request by wangra-google**
[mesa] Update freedreno python script to fix a warning of the generated header

- we dont really care a2xx gpus, and the default value of ~0 is not correct anyways, so set the value to 4294967295 to avoid the warning the prevent compiling the header

### CODE REVIEW

File: third_party/mesa/src/freedreno/common/freedreno_devices.py [hunk 1]
```diff
@@ -204,7 +204,7 @@ def __str__(self):
         gmem_align_w = 32,  gmem_align_h = 32,
         tile_align_w = 32,  tile_align_h = 32,
         tile_max_w   = 512,
-        tile_max_h   = ~0, # TODO
+        tile_max_h   = 0, #GOOGLE, change this to 0 to fix a warning in generated header, original value: ~0, # TODO
```
Comment (wangra-google) [inline third_party/mesa/src/freedreno/common/freedreno_devices.py:207]: i am using the generated header in Dive to check the gpu id (will submit in a different PR), and because of the warning, the compiling failed.

Comment (wangra-google) [reply third_party/mesa/src/freedreno/common/freedreno_devices.py:207]: in the generated header, tile_max_h has a type of uint32_t, and ~0 will generate -1, so the warning is implicit converting from int to uint32_t. without my change, it also has #TODO (so i assume it is a temp one), and i assume eventually someone from mesa side will update the value to a correct one.

Comment (shanminchao) [reply third_party/mesa/src/freedreno/common/freedreno_devices.py:207]: Setting the max tile height to 0 doesn't cause any issues?

Comment (wangra-google) [reply third_party/mesa/src/freedreno/common/freedreno_devices.py:207]: no, it is not used for now. `~0` is already a broken value

Comment (shanminchao) [reply third_party/mesa/src/freedreno/common/freedreno_devices.py:207]: Wait, doesn't setting a -1 to a uint32_t essentially sets it to UINT32_MAX? In that case, that means setting an unbounded max tile height?

Comment (wangra-google) [reply third_party/mesa/src/freedreno/common/freedreno_devices.py]: change the value to 4294967295 which is UINT32_MAX, so should be safe now :)

Comment (shanminchao) [reply third_party/mesa/src/freedreno/common/freedreno_devices.py]: Nitpicky: Is it possible to set it to 0xffffffff instead or  2^32–1?
Seems less error prone in case you get one of those digits wrong.
Nitpicking because eventually we'll upstream this change, so want to make sure we do the cleanest job possible.

Comment (wangra-google) [reply third_party/mesa/src/freedreno/common/freedreno_devices.py]: updated the file as suggested. :)

**Review by shanminchao — APPROVED** [review]
